doc/nix-local-build.rst

```diff
@@ -194,10 +194,10 @@ are not currently built on a per-component basis.
 Where are my build products?
 ----------------------------
 
-A major deficiency in the current implementation of v2-build is that
+A major deficiency in the current implementation of build is that
```

Comment:
```suggestion
A major deficiency in the current implementation of ``cabal build`` is that
```
Looks a bit better but it's up to you. Likewise the next change below.

Reply:
OK, done.